docs/quickstart.md

```diff
@@ -0,0 +1,169 @@
+###Use your existing Spark installation
+You can quickly check the functionality of SnappyData even with your existing Spark 2.0 installation. 
+Only pre-requisites are you should have Spark Version 2.0 installed. Preferably you should have at least 4GB of RAM for the application.
+
+
+
+```scala
+$ cd <Spark_Install_dir>
+$ ./bin/spark-shell --driver-memory 4g --packages "SnappyDataInc:snappydata:0.6.2-s_2.11"
+```
+This will open a Spark shell and download the relevant SnappyData files to your local machine.
+Read and go through the code snippet one by one and copy them to Spark shell to execute.
+
+* Start a SparkSesion in local mode
+```scala
+scala> import org.apache.spark.sql.SparkSession
+
+scala> val spark = SparkSession.builder.master("local[*]").appName("spark, " +
+                   "Snappy Quick Start").getOrCreate()
+```
+
+* Define a of helper function "benchmark", which will give us an average time of a query after doing initial warmups.
+```scala
+scala>  def benchmark(name: String, times: Int = 10, warmups: Int = 2)(f: => Unit) {
+         for (i <- 1 to warmups) {
+           f
+         }
+         val startTime = System.nanoTime
+         for (i <- 1 to times) {
+           f
+         }
+         val endTime = System.nanoTime
+         println(s"Average time taken in $name for $times runs: " +
+           (endTime - startTime).toDouble / (times * 1000000.0) + " millis")
+       }
+
+```
+* Create a DataFrame and temp table using Spark's range method. Cache it in Spark to get optimal performance.
+  This will create a DataFrame of 100 million records.
+```scala
+scala> var df = spark.range(100000000).selectExpr("id", "floor(rand() * 10000) as k")
+scala> df.cache
+scala> df.createOrReplaceTempView("sparkCacheTable")
+```
+
+* Now run a query and to check the performance. The queries is using average of a field without any where clause.
+This will ensure it touches all records while scanning.
+```scala
+scala> benchmark("Spark perf") { spark.sql("select avg(k) from sparkCacheTable").collect() }
+
+scala> spark.sql("select avg(k) from sparkCacheTable").show()
+
+scala> benchmark("Spark perf") {spark.sql("select avg(k) from sparkCacheTable group by (id%100)").collect}
+
+scala> spark.sql("select avg(k) from sparkCacheTable group by (id%100)").show
+
+```
+* Clean up the JVM. This will ensure all in memory artifacts for Spark is cleaned up.
+```scala
+scala> df.unpersist()
+scala> System.gc()
+scala> System.runFinalization()
+```
+
+* Create a SnappyContext. A SnappyContext extends Spark's [[org.apache.spark.sql.SQLContext]] to work with Row and Columnar tables.
```

Comment:
We had decided on using "column" and not "columnar" for terminology. Please replace with "column" throughout docs/code and in this changeset.